processes-endpoint.md

```diff
@@ -314,6 +314,23 @@ This endpoint will return all the different file types for the given process.  T
 
 Returns a single file of the specified process with the provided type (the type matches on the dspace.process.filetype metadata field of the bitstream)
 
+## Search processes
+**GET /api/system/processes/search/byProperty**
+
+This supports a basic search of the processes
+
+The supported parameters are:
+* page, size [see pagination](README.md#Pagination)
+* sort, options are startTime, endTime
+* eperson: optional, the uuid of the eperson who started the process. If not specified, all processes will be returned
+* script: optional, limit the returned processes to the specified script
+* status: optional, limit the returned processes to the specified status. The possible `status` values are `RUNNING`, `COMPLETED` and `FAILED`
```

Comment:
For consistency, we might want to name these the same as the properties returned by `GET /api/system/processes/<:process-id>` (see above in this same file).

In that endpoint, these are named:
* `userId` = eperson's UUID
* `scriptName` = name of script
* `processStatus` = status of process

So, it may be much clearer if we named these parameters the same as the returned JSON.  It might also simplify things for the client side.